backend/graphql/schema.graphql

```diff
@@ -106,6 +153,42 @@ type User {
     email_verified_at: DateTime
     roles: [Role!]! @belongsToMany
     permissions: [Permission!] @belongsToMany
+    profile_metadata: ProfileMetadata
+}
+
+type ProfileMetadataSocialMedia {
```

Comment:
I did this to the types but inputs are still troubling because renaming `UpdateProfileMetadataMailingAddressInput` to `UpdateProfileMetadata_MailingAddressInput` has ramifications that affect naming convention throughout. For example, this can mean that `UpdateProfileMetadataInput` is renamed to `UpdateProfileMetadata_Input` to be consistent, which feels awkward.

Reply:
This is a good point.. I'm going to let that rattle around and maybe look at some other graphql apis to see what they do.

Reply:
I've implemented a decent compromise: since this feature is destined to be scrapped and re-done later, and because we don't anticipate these types interfering with features elsewhere in the system, why try to navigate around a threat of complexity in the future that's unlikely and theoretical? Why not just simplify the names? We have an opportunity to keep this simple and I think we should take advantage of that.

* `UpdateProfileMetadata_MailingAddressInput` => `UpdateMailingAddressInput`
* `UpdateProfileMetadataSocialMediaInput` => `UpdateSocialMediaInput`
* `ProfileMetadata_SocialMedia` => `SocialMedia`
* `ProfileMetadata_MailingAddress` => `MailingAddress`

We'll keep the validators as they are since we're having issues getting namespaces working with a nested directory structure at `/backend/app/GraphQL/Validators/ProfileMetadata/`.

What do you think?

Reply:
Love it.  I'll approve